### DISCUSSION TITLE
Error: Untrusted Package

### DISCUSSION THREAD

**Discussion by esdrasspd** [passwordless-lib/fido2-net-lib#522] [Q&A]
Hello, nice team, I work for a banking company in Guatemala and we are implementing PassKey in our system, so we downloaded their public project using the .NET 8 version and the Visual Studio 2022 version 17.9.7 (the most recent). However, we get an error in the package: Microsoft.NET.ILLink.Tasks 8.05 and it says: This package is signed, but by a trusted signer. The same thing also happens with the Microsoft.NET.Sdk.WebAssembly.Pack 8.0.5 package.
![Paquetes no de confianza](https://github.com/passwordless-lib/fido2-net-lib/assets/102131093/97d99e3e-f7a1-4137-803a-303f5f324380)

**Comment by abergs** [comment]
Hey @esdrasspd - were you able to resolve this?

**Reply by esdrasspd** [reply]
Hi, we were never able to resolve this error.

**Reply by FrankSzendzielarz** [reply]
This has something to do with nuget config here %appdata%\NuGet\NuGet.Config under the trustedSigners node. Either add the missing name or remove the node
